src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+function createPromise1() {
+  const logo = document.querySelector('.logo');
+
+  const resolver = (resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
```

Comment:
it can be simplified
```suggestion
    logo.addEventListener('click', resolve);
```